Assets/Plugins/iOS/Shopify/BuyTests/BuySerializableTests.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  BuySerializableTests.swift
+//  UnityBuySDK
+//
+//  Created by Shopify.
+//  Copyright © 2017 Shopify Inc. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import XCTest
+import PassKit
+import ProductName
+
+class BuySerializableTests: XCTestCase {
+    
+    let emailAddress = "test_email@shopify.com"
+    let firstName    = "first name"
+    let lastName     = "last name"
+    let phone        = "123-456-7890"
+    let address1     = "80 Spadina Ave"
```

Comment:
So just for documentations sake I wanted to document our IRL convo.

Since Apple encodes `Address1` and `Address2` into `street` we're gonna attempt to parse `Address1` and `Address2`from `street`